File: codeforces/round98/c.cpp
#include <iostream>
#include <algorithm>

using namespace std;

struct event{
  int start, end;
} events[100000];

int n;

bool cmp(event a, event b){
  return a.start < b.start;
}

int main(){

  cin >> n;

  for(int i = 0; i < n; i ++){
    cin >> events[i].start >> events[i].end;
  }

  sort(events, events + n, cmp);


  // for(int i = 0; i < n; i ++){
  //   cout << events[i].start << " " << events[i].end << endl;
  // }
 
  int answer = 0;
  int maxend = events[0].end;

  for(int i = 1; i < n; i ++){
    if(events[i].end < maxend){
      answer ++;
    }else{
      maxend = events[i].end;
    }
  }

  cout << answer << endl;

  return 0;
}

File: codeforces/round98/c.py
def comp(x, y):
    if x[0] > y[0]:
        return 1
    elif x[0] < y[0]:
        return -1
    
    return 0


if __name__ == '__main__':
    
    n = int(raw_input())

    events = []

    for i in xrange(n):
        l = raw_input().split()
        events.append((int(l[0]), int(l[1])))

    events.sort(comp)

    #print events

    answer = 0

    for i in xrange(1, n):
        if events[i-1][0] < events[i][0] and events[i-1][1] > events[i][1]:
            events[i] = events[i-1]
            answer += 1
        
    print answer
